ArduCopter/esc_calibration.cpp

```diff
@@ -104,6 +104,17 @@ void Copter::esc_calibration_passthrough()
     // disable safety if requested
     BoardConfig.init_safety();
 
+    // wait for safety switch to be pressed
+    while (hal.util->safety_switch_state() == AP_HAL::Util::SAFETY_DISARMED) {
+        bool printed_msg = false;
```

Comment:
Txs for the review.  It didn't have a 1hz message before though, so technically it's scope creep.  There's no way for the user to see it anyway because Mavlink doesn't work during the ESC calibration (at least in my tests).